portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -56,12 +55,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String text = request.getParameter("text-input");
     long timestamp = System.currentTimeMillis();
```

Comment:
uber nit - you're only going to need the timestamp if the text is not null or empty. Consider moving the timestamp variable into that conditional block.

Reply:
noted-updated!